BraintreeCore/src/main/java/com/braintreepayments/api/ApiClient.kt

```diff
@@ -35,21 +29,46 @@ class ApiClient constructor(private val braintreeClient: BraintreeClient) {
         try {
             val body = paymentMethod.buildJSON().toString()
             braintreeClient.sendPOST(url, body) { responseBody, httpError ->
-                if (responseBody != null) {
```

Comment:
Should we be sending the equivalent of the 2 graphQL analytic events above, for REST as well?

Reply:
Not a bad idea to see if there are some merchants out there with RESTful card tokenization configs left.

Reply:
Theoretically though the REST tokenization should be highly unlikely, unless there's some legacy merchant with REST tokenization switched on.

Reply:
That's a great question. I wonder if we need to support REST at all anymore, or if all merchants are enabled for GraphQL.